section-09-differential-testing/model-package/MANIFEST.in

```diff
@@ -0,0 +1,18 @@
+include *.txt
+include *.md
+include *.pkl
+recursive-include ./src/regression_model/*
+
+include src/regression_model/datasets/train.csv
```

Comment:
It was necessary to introduce an additional layer to the structure (i.e. the `src` directory) so that in the tests we could distinguish between the present files and an installed package (whether a local installed package via pip install -e or fetched from pypi)